vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -117,7 +152,9 @@ def create_weights(self, layer: torch.nn.Module,
                        **extra_weight_attrs):
         """
         Use the CompressedTensorsScheme associated with each layer to create 
-        the necessary parameters for the layer.
+        the necessary parameters for the layer. See LinearMethodBase for param
```

Comment:
nit - make PR cleaner (dont touch this line)

Reply:
The comment was expanded.